repo/packages/K/kubernetes/3/package.json

```diff
@@ -11,7 +11,7 @@
     "icon-large": "https://downloads.mesosphere.com/kubernetes/artifacts/images/k8s-256x256.png"
   },
   "tags": ["mesosphere", "framework", "pod", "label", "paas"],
-  "preInstallNotes": "In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 1 node with 1 CPU share and 1 GB of RAM available for use by the Kubernetes service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
+  "preInstallNotes": "In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 1 node with 1 CPU share and 1 GB of RAM available for use by the Kubernetes service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.\n\nWARNING: The standard configuration launches an integrated etcd WITHOUT PERSISTENCE.\n         For production follow http://docs.mesosphere.com/services/kubernetes/#install\n         to configure kubernetes with a highly available etcd cluster.\n",
```

Comment:
Easy way to prove: scale the kubernetes app down to 0 and up to 1 in Marathon. BOOM. Two frameworks registered.